### PR TITLE
Update coffee shops (also, add Recovery Room)

### PR DESCRIPTION
**Pull request by rick**
Added steadfast coffee. I mean, it's kinda overrun with twitterati and skillery people, but the coffee is good and it's possible to hack from here; more legit than the red :bike:.

Whatevs.

/cc @jfryman @jameswhite @azizshamim @bval @awebneck ... ?  (we probably need an @websages team for this shit)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,13 +101,14 @@
...
- - [J&J's Market & Cafe][jjs] -- Vanderbilt area, decent-ish coffee, good tea selection, quiet, good atmosphere, decent internet, only small snack food (go next door to Noshville to eat).
+ - [J&J's Market & Cafe][jjs] -- Vanderbilt area, decent-ish coffee, good tea selection, quiet, good atmosphere, decent internet, only small snack food (go next door to Noshville to eat). It's days are numbered, due to construction. :crying_cat_face:
  - [Ugly Mugs][uglymugs] -- decent internet, decent-ish coffee, nice atmosphere; next to Jeni's ice cream (zomg.)
  - [Portland Brew][portlandbrew] (multiple locations) -- decent coffee; ~~the 12South location is managed by Sra (pronounced "shraa"), who's a friend and former neighbor of @rick -- tell Sra I sent ya.~~ Sorry, bro, Sra split. :dash:
- - [Red Bicycle][redbicycle] -- If you're over in Germantown this is a coffee shop. Not highly recommended, but passable in a tight spot.
+ - [Red Bicycle][redbicycle] -- If you're over in Germantown this is a coffee shop. Not highly recommended, but passable in a tight spot. Go to Steadfast instead.
```
Comment (jameswhite) [inline README.md:111]: I like the Red Bicycle's crepes. Does Steadfast have crepes?

Comment (jameswhite) [inline README.md:111]: And now I want crepes. THANKS A LOT @rick

Comment (rick) [inline README.md:111]: Steadfast has food, but not crepes that I saw.

**Comment by rick** [comment]
Added linkage to Recovery Room, fwiw.

File: README.md [hunk 2]
```diff
@@ -101,13 +101,14 @@
...
  - [Frothy Monkey][frothy] -- long a staple of the 12South area, the newly opened downtown location is a block away from the GitHub coworking space, stays open til 9pm (10pm on friday/saturday), and is easily the best coffee shop downtown. Oh, also decent beers, wine, and cocktails. :cocktail:
  - [Crema][crema] -- decent coffee, near Emma HQ
  - [Cafe Coco's][cafecoco] -- 24x7 coffee, tea, beer (well, during legal hours), food, etc.  Indoors and outdoors, exceedingly diverse clientelle, internet can be shitty; take no pride in / give zero fucks about crafting a latte, but they roast their own coffee.  My personal favorite for getting away and hacking.
- - [J&J's Market & Cafe][jjs] -- Vanderbilt area, decent-ish coffee, good tea selection, quiet, good atmosphere, decent internet, only small snack food (go next door to Noshville to eat).
+ - [J&J's Market & Cafe][jjs] -- Vanderbilt area, decent-ish coffee, good tea selection, quiet, good atmosphere, decent internet, only small snack food (go next door to Noshville to eat). It's days are numbered, due to construction. :crying_cat_face:
```
Comment (azizshamim) [inline README.md:108]: > It's days are numbered, due to construction. :crying_cat_face:

WHAT?

Comment (jameswhite) [inline README.md:108]: @azizshamim the entire block with JJ's and Noshville was bought by a condominium developer.

**Comment by bval** [comment]
:+1: the coffee is much better at Steadfast than :red_circle: :bike:. There's a Barista Parlor opening at 4th and Monroe in a few weeks as well.

**Comment by jameswhite** [comment]
CREPES @bval CREPES

**Comment by rick** [comment]
I totally walked past the Germantown Barista Parlor site today.
Gentrification is almost complete.

On Thu, Oct 22, 2015 at 5:16 PM, Brandon Valentine <notifications@github.com

**Pull request merged**